common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -96,6 +96,16 @@ function (HTML5Video, Resizer) {
             $(window).on('unload', state.saveState);
 
             if (!state.isFlashMode() && state.speed != '1.0') {
+
+                // Work around a bug in the Youtube API that causes videos to
+                // play at normal speed rather than at the configured speed in
+                // Safari.  Setting the playback rate to 1.0 *after* playing
+                // started and then to the actual value tricks the player into
+                // picking up the speed setting.
+                if (state.browserIsSafari && state.isYoutubeType()) {
+                    state.videoPlayer.setPlaybackRate(1.0, false);
```

Comment:
In Flash mode, the player has to reload the video to select a different speed, and the bug does not occur.